.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,80 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+Thank you for opening an issue. Please note that we try to keep the issue
+tracker reserved for bug reports and feature requests. For general usage
+questions, please see:
+https://github.com/digitalocean/terraform-provider-digitalocean/discussions
```

Comment:
I'm good with this, but worth noting that we haven't really used the discussions feature for any of our repos yet. Any opinions from @digitalocean/api-cli

Reply:
I meat to point that out too. I figured this would be a good way to start?